kubelet/datadog_checks/kubelet/prometheus.py

```diff
@@ -285,14 +285,19 @@ def _process_container_metric(self, type, metric_name, metric, scraper_config, l
             if self.pod_list_utils.is_excluded(c_id, pod_uid):
                 continue
 
-            tags = scraper_config['custom_tags'][:]
-            tags += tagger.tag(replace_container_rt_prefix(c_id), tagger.HIGH) or []
+            tags = tagger.tag(replace_container_rt_prefix(c_id), tagger.HIGH) or None
```

Comment:
```suggestion
            tags = tagger.tag(replace_container_rt_prefix(c_id), tagger.HIGH)
```
- `or None` seems not needed